### DISCUSSION TITLE
Few Colors Suggestions

### DISCUSSION THREAD

**Discussion by akiarostami** [Siumauricio/rippleui#102] [Ideas]
Hello,

Let me start by saying that I absolutely love how clean and intuitive Ripple UI is. Thank you for doing this.

I am missing one thing in your Base colors: Info. In your default palette, your primary color is Blue, which can be used for Info. But if one changes the primary color, then there's no option for the "Info" family. It gets more challenging as your colors (or Tailwind colors) are not available in components such as Badge.

Besides that, I would have loved to be able to use Tailwind colors with your components (e.g. "badge-orange-500").

Lastly, it seems that customized primary color is not applied to some components. For example, btn-solid-primary still uses the default (blue), and not my customized primary color.

**Comment by Siumauricio** [comment]
Hello @akiarostami Thanks!! and that's correct, in some specific classes we don't use primary colors, instead we use the pallete colors, why?, it's very difficult to mix colors and try to have a very good perception of each class could be hard to mantain, so that's the reasson why in some classes we use pallete colors instead primary.

When you refer you can't use Tailwind Colors you mean, you cannot use in your project ? you should have all the tailwind colors available without any issue, here is a example, you can use any color

```html
<button class="btn !bg-red-600">Hello World!</button>
```

I have been thinking about implement customization in level-components which means you can apply a few properties in each component something like this, we have a discussion here https://github.com/Siumauricio/rippleui/issues/97

```js
module.exports = {
	/** @type {import('rippleui').Config} */
	rippleui: {
		themes: [
			{
				themeName: "light",
				colorScheme: "light",
				colors: {
					Button:{
					  // Only a few properties 
					   backgroundColor:"#432443",
					   backgroundHover: "#32345"
					
					}
				},
			},
		],
	},
}
```

**Comment by akiarostami** [comment]
Hell @Siumauricio and thank  you for all the information.

I think component-level customization would be great (even though I'm already doing that using Tailwind layers).

I still would love to see color "Info" added to the base color palette, next to Warning, Error, and Success. :)
